MAUI/Segmented-Control/display-mode.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title: Display options in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Display options in Syncfusion .NET MAUI Segmented(SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Display options in .NET MAUI Segmented (SfSegmented) control
+
+Depending on the application, different scenarios may require icons, text, or a combination of both for effective communication. The segmented control supports these three options.
```

Comment:
remove this 
The segmented control supports these three options.

Reply:
removed